base/linalg/symmetric.jl

```diff
@@ -57,9 +57,9 @@ ctranspose(A::Hermitian) = A
 trace(A::Hermitian) = real(trace(A.data))
 
 #tril/triu
-tril(A::Hermitian,k::Integer=0) = tril(A.data,k)
+tril(A::Hermitian,k::Integer=0) = triu(A.data,-k)'
```

Comment:
I think these need to depend on `uplo`, since sometimes the important data is stored in the upper triangle, sometimes in the lower